src/runtime_src/core/common/info_vmr.cpp

```diff
@@ -39,27 +39,24 @@ pretty_label(std::string label)
 
 } //unnamed namespace
 
-namespace xrt_core { 
-namespace vmr {
+namespace xrt_core::vmr {
 
 ptree_type
 vmr_info(const xrt_core::device* device)
 {
   ptree_type pt_vmr_status_array;
   ptree_type pt_vmr_stats;
-  std::vector<std::string> vmr_status_raw;
-  try {
-    vmr_status_raw = xrt_core::device_query<xq::vmr_status>(device);
-    auto vmr_version = xrt_core::device_query<xq::extended_vmr_status>(device);
-    vmr_status_raw.insert(vmr_status_raw.begin(), vmr_version.begin(), vmr_version.end());
-  }
-  catch (const xq::exception&) {
-    // only available for versal
+  std::vector<std::string> empty_vector;
+  auto vmr_status = xrt_core::device_query_default<xq::vmr_status>(device, empty_vector);
+  auto vmr_version = xrt_core::device_query_default<xq::extended_vmr_status>(device, empty_vector);
```

Comment:
```suggestion
  auto vmr_status = xrt_core::device_query_default<xq::vmr_status>(device, {});
  auto vmr_version = xrt_core::device_query_default<xq::extended_vmr_status>(device, {});
```
This should work given that QueryRequestType::result_type is std::vector.  Please change in other places also if same pattern is used.